skabase/SKALogger/SKALogger.py

```diff
@@ -80,18 +72,18 @@ def init_device(self):
 
     def write_storageLoggingLevel(self, value):
         self._storage_logging_level = value
-        if self._storage_logging_level == int(tango.LogLevel.LOG_FATAL):
-            logger.setLevel(logging.FATAL)
-        elif self._storage_logging_level == int(tango.LogLevel.LOG_ERROR):
-            logger.setLevel(logging.ERROR)
-        elif self._storage_logging_level == int(tango.LogLevel.LOG_WARNING):
-            logger.setLevel(logging.WARNING)
-        elif self._storage_logging_level == int(tango.LogLevel.LOG_INFO):
-            logger.setLevel(logging.INFO)
-        elif self._storage_logging_level == int(tango.LogLevel.LOG_DEBUG):
-            logger.setLevel(logging.DEBUG)
-        else:
-            logger.setLevel(logging.DEBUG)
+        # if self._storage_logging_level == int(tango.LogLevel.LOG_FATAL):
```

Comment:
Where is this all commented out?  Another behavioural change.

Reply:
Reverted back the changes. This is addressed in other pull request.